### DISCUSSION TITLE
Content for pdf response helper

### DISCUSSION THREAD

**Discussion by discoverlance-com** [sergiodxa/remix-utils#115] [Q&A] [answered]
What type of or example of content does the pdf response helper accept? I tried using a blob but that failed so I am not sure what content to add there in other to use the pdf response.

My example: 

```ts
export let loader: LoaderFunction = async ({ request }) => {
  return pdf(
    new Blob(
      ["<html> <body> <h1>Hello World</h1></body> </html>"],
      { type: "text/html" }
    )
  );
};

```

The above example I tried does not work and the pdf gives an error: `Failed to load PDF document.`

**Comment by sergiodxa** [comment, marked as answer]
You need to pass an actual PDF file, the helper doesn’t convert HTML to PDF, it sets the correct Content-Type.
